karma.conf.js

```diff
@@ -53,12 +65,7 @@ module.exports = function(config) {
     let browsers = [
         'ChromeHeadless',
         'Chrome',
-        'Safari',
```

Comment:
We need to schedule some time internally to get tests running correctly on all the browsers that we want.

Reply:
What's going on with the Safari tests?

Reply:
Since we were always running through grunt safari was never run so our unit tests currently fail for various reasons. Not sure that it's anything to be concerned about, just something we want to rectify eventually.